Upload code Condition Tab

anabelle2134 opened the pull request:
Create CarForSaleConditionTabComponent class
Create CarForSaleConditionTabComponentTest class

anabelle2134 committed:
Upload code Condition Tab

Create CarForSaleConditionTabComponent class
Create CarForSaleConditionTabComponentTest class

namnguyen-itvn commented:
@doanhoangtri please review this PR

doanhoangtri commented:
- Coding convention: fine
- test run OK
- Have not used Data Driven
- no Function Assert to verify TestCase

namnguyen-itvn reviewed (changes requested):
pls update the review as mention from reviewer

anabelle2134 committed:
Create newCarForSale.class
Create CarForSaleConditiontabComponent.class
Create CarForSaleTests.class
Update Assert

anabelle2134 commented:
update have been pushed

doanhoangtri commented on condition/src/test/java/com/example/test/pages/KBBMOOCProject/CarForSaleConditionTabComponent.java:
```diff
@@ -0,0 +1,84 @@
+     */
+    public boolean checkOptionIsDisplayedCorrect() {
+        keyword.scrollToElement(titleModuleYourSearch);
+        WebElement optionConditionYourSearch = keyword.findElement(By.xpath(
```
should write the web element locator in separate line then call into a method/function

doanhoangtri reviewed (commented):
should write the web element locator in separate line then call into a method/function
